RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -5792,3 +5797,140 @@ int auto_reserveCraftAmount(item orig_it)
 	}
 	return inner(orig_it);
 }
+
+
+// This just does the math for comparing vs. the Cap. If no cap is set then ML is virtually unlimited.
+int remainingMLToCap()
+{
+	int MLToCap = 0;
+
+	if(get_property("auto_MLSafetyLimit") == "")
+	{
+		MLToCap = 999999;
+	}
+	else if(monster_level_adjustment() < get_property("auto_MLSafetyLimit"))
+	{
+		MLToCap = get_property("auto_MLSafetyLimit") - monster_level_adjustment();
+	}
+
+	return MLToCap;
+}
+
+// Gives us the number we need when comparing to a desired ML or entering a value into a maximizer string.
+int auto_convertDesiredML(int DML)
+{
+
+	int DesiredML = get_property("auto_MLSafetyLimit");
+
+	if((get_property("auto_MLSafetyLimit") == "") || (DML < get_property("auto_MLSafetyLimit")))
```

Comment:
I think you need `to_int(get_property("auto_MLSafetyLimit"))` dont you? I dont think ash will auto convert string to int for you, though int to string works.

Reply:
I shouldn't since it is stored as an int in the header, but after examining some others (ashtonlimit) I see it is converted to int when doing a comparison. So I modified it to convert to int. Better safe than sorry.